src/models/Request.ts

```diff
@@ -48,43 +50,45 @@ export default class StakeRequest extends Comparable<StakeRequest> {
   public updatedAt?: Date;
 
   public constructor(
-    stakeRequestHash: string,
+    requestHash: string,
+    requestType: string,
     blockNumber: BigNumber,
     amount?: BigNumber,
     beneficiary?: string,
```

Comment:
why these parameters are optional?

Reply:
There are cases where stakeRequest object is created with mandatory arguments. It was existing logic. Also in multiple tests, stakeRequest is created with mandatory arguments.
e.g. method: updateMessageHashInRequestRepository in AcceptStakerequestService.